2021-11-18 11:15:17,696 INFO    MainThread:206740 [wandb_setup.py:_flush():71] setting env: {}
2021-11-18 11:15:17,696 INFO    MainThread:206740 [wandb_setup.py:_flush():71] setting login settings: {}
2021-11-18 11:15:17,697 INFO    MainThread:206740 [wandb_init.py:_log_setup():357] Logging user logs to /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20211118_111517-imvyhh13/logs/debug.log
2021-11-18 11:15:17,697 INFO    MainThread:206740 [wandb_init.py:_log_setup():358] Logging internal logs to /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20211118_111517-imvyhh13/logs/debug-internal.log
2021-11-18 11:15:17,697 INFO    MainThread:206740 [wandb_init.py:init():390] calling init triggers
2021-11-18 11:15:17,697 INFO    MainThread:206740 [wandb_init.py:init():395] wandb.init called with sweep_config: {}
config: {'width': 25, 'height': 25, 'num_agents': 1, 'max_num_cities': 2, 'seed': 1, 'grid_mode': False, 'max_rails_between_cities': 2, 'max_rails_in_city': 4, 'malfunction_rate': 0, 'min_duration': 20, 'max_duration': 50, 'observation_builder': 'GraphObsForRailEnv', 'predictor': 'ShortestPathPredictorForRailEnv', 'view_semiwidth': 7, 'view_height': 30, 'offset': 25, 'observation_depth': 3, 'num_episodes': 1000, 'start_epoch': 0, 'max_steps': 300, 'eps': 1, 'debug_print': False, 'load_memory': False, 'evaluation_episodes': 3, 'render': False, 'evaluation_interval': 10, 'save_model_interval': 50, 'start_lr_decay': 150, 'eps_decay': 0.999, 'learning_rate': 0.005, 'learning_rate_decay': 1.0, 'learning_rate_decay_policy': 1.0, 'run_title': 'first_run', 'wandb_project_name': 'wandb_default', 'model_path': 'test_results/', 'model_name': '', 'gat_layers': 3, 'flow': 'source_to_target', 'resume_weights': True, 'batch_norm': False, 'dropout_rate': 0.8, 'attention_heads': 4, 'batch_size': 128, 'use_stop_action': False, 'done_reward': 0, 'deadlock_reward': -1000, 'reward_scaling': 0.1}
2021-11-18 11:15:17,697 INFO    MainThread:206740 [wandb_init.py:init():435] starting backend
2021-11-18 11:15:17,697 INFO    MainThread:206740 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-11-18 11:15:17,707 INFO    MainThread:206740 [backend.py:ensure_launched():199] starting backend process...
2021-11-18 11:15:17,716 INFO    MainThread:206740 [backend.py:ensure_launched():204] started backend process with pid: 206922
2021-11-18 11:15:17,717 INFO    MainThread:206740 [wandb_init.py:init():444] backend started and connected
2021-11-18 11:15:17,719 INFO    MainThread:206740 [wandb_init.py:init():503] updated telemetry
2021-11-18 11:15:17,739 INFO    MainThread:206740 [wandb_init.py:init():533] communicating current version
2021-11-18 11:15:18,547 INFO    MainThread:206740 [wandb_init.py:init():538] got version response 
2021-11-18 11:15:18,547 INFO    MainThread:206740 [wandb_init.py:init():548] communicating run to backend with 30 second timeout
2021-11-18 11:15:48,548 INFO    MainThread:206740 [interface_queue.py:_communicate_run():336] couldn't get run from backend
2021-11-18 11:15:48,548 ERROR   MainThread:206740 [wandb_init.py:init():553] backend process timed out
2021-11-18 11:15:48,548 ERROR   MainThread:206740 [wandb_init.py:init():562] encountered error: Error communicating with wandb process
try: wandb.init(settings=wandb.Settings(start_method='fork'))
or:  wandb.init(settings=wandb.Settings(start_method='thread'))
For more info see: https://docs.wandb.ai/library/init#init-start-error
2021-11-18 11:19:25,510 WARNING MainThread:206740 [wandb_init.py:init():891] interrupted
Traceback (most recent call last):
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 870, in init
    run = wi.init()
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 566, in init
    backend.cleanup()
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/wandb/sdk/backend/backend.py", line 230, in cleanup
    self.wandb_process.join()
  File "/home/runnphoenix/anaconda3/lib/python3.8/multiprocessing/process.py", line 149, in join
    res = self._popen.wait(timeout)
  File "/home/runnphoenix/anaconda3/lib/python3.8/multiprocessing/popen_fork.py", line 47, in wait
    return self.poll(os.WNOHANG if timeout == 0.0 else 0)
  File "/home/runnphoenix/anaconda3/lib/python3.8/multiprocessing/popen_fork.py", line 27, in poll
    pid, sts = os.waitpid(self.pid, flag)
KeyboardInterrupt
